SemanticMetaTags.php

```diff
@@ -103,7 +88,6 @@ public static function onExtensionFunction() {
 			new Options( $configuration )
 		);
 
-		$hookRegistry->register();
```

Comment:
You need this line otherwise the class doesn't know what to do.

Reply:
SRF is apparently using a variant `self::registerHooks();` however I did actually think of keeping this.